### ISSUE TITLE
Make README clearer on `.medium_list`

### ISSUE THREAD

**Issue by Utagai** [Utagai/spice#30]
Issue #29 makes me think, upon looking at the README, that we are wrong/unclear about how to use a `MediumList`. We write the following in the README:

```
...
mlist.anime/manga_list, dictionary containing 5 keys whose values contain 5 lists according to the key:
...
```

But this is pretty misleading, since what we actually want is:

```
...
mlist.medium_list, dictionary containing 5 keys whose values contain 5 lists according to the key:
...
```

This is also partly my fault for not updating the source doc strings, found [here](https://github.com/Utagai/spice/blob/master/spice_api/objects.py#L475-L480). The change to using `medium_list` was done because there's no need to differentiate between medium lists.

In either case, its also worth considering just making `MediumList` objects iterable and save the hassle. Lots of this code was written when I was an infant `python` developer, and most likely, turning `MediumList` into a true `list` wrapper is probably the best play.

**Comment by eshrh** [comment]
Yeah, good idea, i'm working on it!

**Comment by eshrh** [comment]
for now, though, changing the readme should fix #29 immediately
#31 does it
